apps/emqx_bridge_datalayers/README.md

```diff
@@ -0,0 +1,43 @@
+# EMQX Datalayers Bridge
+
+[Datalayers](https://docs.datalayers.cn/datalayers/latest/) is a multimodal,
```

Comment:
this link is 404

Reply:
may be just https://datalayers.io/
EN doc site is empty

Reply:
Their Product team told me they only have the Chinese site for now  🤒